spatialmath/orientationVector.go

```diff
@@ -20,19 +20,19 @@ import (
 // point, and the plane defined by the origin, the rx,ry,rz point, and the new local Z axis. So if theta is kept at
 // zero as the north/south pole is circled, the Roll will correct itself to remain in-line.
 type OrientationVector struct {
-	Theta float64 `json:"th"`
-	OX    float64 `json:"x"`
-	OY    float64 `json:"y"`
-	OZ    float64 `json:"z"`
+	Theta float64 `json:"th_rads"`
+	OX    float64 `json:"x_mm"`
+	OY    float64 `json:"y_mm"`
+	OZ    float64 `json:"z_mm"`
 }
 
 // OrientationVectorDegrees is the orientation vector between two objects, but expressed in degrees rather than radians.
 // Because protobuf Pose is in degrees, this is necessary.
 type OrientationVectorDegrees struct {
-	Theta float64 `json:"th"`
-	OX    float64 `json:"x"`
-	OY    float64 `json:"y"`
-	OZ    float64 `json:"z"`
+	Theta float64 `json:"th_rads"`
```

Comment:
This is in degrees, not rads

Reply:
oh gosh, clearly misread the comment here :sweat_smile: